tests/Aryacoin/TWAddressTests.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+//
+
+#include "../interface/TWTestUtilities.h"
+#include "Bitcoin/Address.h"
+#include "PrivateKey.h"
+#include "PublicKey.h"
+#include "HexCoding.h"
+#include "Coin.h"
+#include <gtest/gtest.h>
+#include <TrustWalletCore/TWAnyAddress.h>
+
+using namespace TW;
+
+const char *AYA_PRIVATE_KEY = "403769C2CC212A04B13830EA108DCABF8EC87DDD1DCBB1E9BB8F6E539649DE8E";
```

Comment:
Could you please use preferred `type* id` style instead of `type *id`?

Reply:
Updated in [2543356](https://github.com/trustwallet/wallet-core/pull/1013/commits/da7e1c62bff7d788c9c369d0d5a426c66140d283)